packages/manager/modules/pci/src/projects/project/sidebar/translations/Messages_fr_FR.json

```diff
@@ -21,7 +21,7 @@
   "cloud_sidebar_management_interface_cloud_shell": "Cloud Shell",
   "cloud_sidebar_project_management": "Project management",
   "cloud_sidebar_project_management_users": "Utilisateurs",
-  "cloud_sidebar_project_management_quota_region": "Quota & Région",
+  "cloud_sidebar_project_management_quota_location": "Quota & Localisation",
```

Comment:
```suggestion
  "cloud_sidebar_project_management_quota_location": "Quota & localisation",
```
?

Reply:
I've asked @fchabrolle which told me to stay like this in order to make consistency.